diff --git a/node_modules/freeverse-marketsigner-js/dist/main.js b/node_modules/freeverse-marketsigner-js/dist/main.js
index 816f57c..52b29ec 100644
--- a/node_modules/freeverse-marketsigner-js/dist/main.js
+++ b/node_modules/freeverse-marketsigner-js/dist/main.js
@@ -119,13 +119,15 @@ class $b6e421008fa6e70e$var$NativeCryptoPayments {
     finalizeAndWithdraw({ assetTransferData: assetTransferData , signature: signature , from: from  }) {
         return this.paymentsContract.methods.finalizeAndWithdraw(assetTransferData, signature).send({
             from: from,
-            value: 0
+            value: 0,
+            gasLimit: "600000"
         });
     }
     pay({ paymentData: paymentData , signature: signature , from: from  }) {
         return this.paymentsContract.methods.pay(paymentData, signature).send({
             from: from,
-            value: paymentData.amount
+            value: paymentData.amount,
+            gasLimit: "60000"
         });
     }
     finalize({ assetTransferData: assetTransferData , signature: signature , from: from  }) {
@@ -235,7 +237,8 @@ class $c567d869dd32e308$var$ERC20Payments extends $c567d869dd32e308$require$Nati
     pay({ paymentData: paymentData , signature: signature , from: from  }) {
         return this.paymentsContract.methods.pay(paymentData, signature).send({
             from: from,
-            value: 0
+            value: 0,
+            gasLimit: "60000"
         });
     }
     approve({ amount: amount , from: from  }) {
